Add pdfs.hbs [WIP]

cstby opened the pull request:
This is a first shot at issue #117.

A few notes:
- I didn't include the bleed pdf because I didn't see a use case for the normal user. The only time one would need a bleed pdf would be for professional printing, correct? Including the bleed pdf is trivial, so if I'm mistaken in understanding the use case, I'm happy to add it.
- I used the existing styles from `stylesheet-index.css` and avoided adding more code to the css file. This means that added a few manual breaks into the hbs template. I believe this is simpler and more maintainable than editing the css to account for the flexbox spacing.
- I used good judgment on the copy, but I'm not attached to it if we want to change it.


Any and all feedback is welcome!

alifeee commented:
toki a! pona mute tawa sina tan pali sina a

this looks great :)

as the bot says, we can find a preview on <https://liputenpotest-118.surge.sh/> (but because of filesize, the preview does not have PDFs)

when I made the issue, I was just imagining a pretty plaintext-list, but the images and layout makes it look nice :)

a few notes:

- "Color A5 PDF" etc. is in English. It should be TP, e.g., "lipu kule", "lipu tawa ilo sitelen", "lipu pimeja/walo tawa ilo sitelen"
- I don't know that specifying A5/A4 is that helpful (and in some cases like nanpa lili, I believe wrong). This could be left out
- there is not necessarily the three types of PDF for each issue. For example
  - many issues have an extra PDF: the sitelen pona version !
  - some issues do not have print versions, e.g., nanpa kijetesantakalu / kati tiki tu lili
    thus, we should use data from `lipu_ale.yaml` and use conditionals, as we do [on the `lipu` pages](https://github.com/lipu-tenpo/liputenpo.org/blob/c7a5ca061a536af9aaa96c6753f5c700d9c2e798/lipu/lipu.hbs#L30-L38)
- the word "PDF" interspaced with toki pona words seems strange to me. perhaps `lipu` is fine. otherwise, the copy is nice :)

overall this is a wonderful change ! thanks for giving it an attempt :)

I hope you find eleventy to your pleasure.

with changes to the note of my notes, this is easily ready to merge and a good page (almost an easter egg of a page ;)

alifeee reviewed (changes requested):
a final note: I prefer the header to stay `lipu` rather than `lipu pdf` :)

cstby commented:
Thanks again for the quick feedback!

- I changed the copy as you suggested (removing "pdf" and using toki pona instead of Engilsh).
- I also added conditional logic for each pdf type, including adding sitelen pona.

I noticed that the long copy "lipu walo tawa ilo sitelen" used the full width and spilled over onto a second line. My final commit adds some styling changes to make each lipu wider so that each pdf link only uses one line. I think this looks a little better, but it's totally up to you whether you'd prefer not to add any additional CSS (or to go a different direction with the design of this page).

alifeee commented:
the CSS is fine with me!

this looks ready to merge !!

only one thing I should have said before: I think we should put a link at the top of the page that says "download all" or similar (in toki pona), which links to <https://github.com/lipu-tenpo/liputenpo.org/tree/main/pdfs>

after that I will merge it :) thank you for doing this

cstby commented:
Added in that link in a new line in the introduction section!

alifeee commented:
I have made a small change to move the URL to `sona.yaml` (in `_data`), so as to not have implementation-specific (GitHub) stuff in the HTML templates.

thanks a lot for this !! I will merge and it should be visible on the site soon :)
